docs/details/shards/General/Hashed.md

```diff
@@ -0,0 +1,6 @@
+`(Hashed)` takes a single shard or a sequence of shards and hashes the output of the last shard (or the only shard) in this sequence. So this shard can be used to group together certain shards when you want their final output to be hashed.
+
+Like [(`Sub`)](../sub)), `(Hashed)` too uses `(->)` but unlike `(Sub)`, `(Hashed)` doesn't pass its input to all its shards in parallel. The input to `(Hashed)` becomes input to the first shard in the hashed sequence and the output of this shard becomes the input for the next shard in the hashed sequence (following the regular Shards data flow pattern).
```

Comment:
This is confusing. It is the regular flow of any shards to give the input to the first inner shard. `Sub` also doesn't do anything in parallel, certainly not giving the same input to each inner shards.

```clj
5
(| (Math.Add 1)
   (Log) ;; this won't print 5 but 6 as expected
)
;; however, output will be 5 here
```
```

Reply:
Statement corrected.